docs/guides/private-python-packages.rst

```diff
@@ -0,0 +1,108 @@
+Using Private Python Packages with Read the Docs for Business
+=============================================================
+
+Read the Docs uses :doc:`PIP <pip:index>` to install your Python packages,
```

Comment:
We use lowercase "pip" [other places in our docs](https://docs.readthedocs.io/en/stable/guides/specifying-dependencies.html). We should probably be consistent.